doctr/models/classification/vit/pytorch.py

```diff
@@ -148,27 +148,25 @@ def vit_b(pretrained: bool = False, **kwargs: Any) -> VisionTransformer:
     """
 
     return _vit(
-        "vit_b",
+        "vit_s",
         pretrained,
-        d_model=768,
+        d_model=384,
         num_layers=12,
-        num_heads=12,
+        num_heads=6,
         ffd_ratio=4,
-        ignore_keys=["head.weight", "head.bias"],
+        ignore_keys=["2.head.weight", "2.head.bias"],
```

Comment:
Why `2.head.*` ? :thinking:

Reply:
model structure is:
(0) patch embedding
(1) encoder block
(2) classifier head 

the head.weight was only a placeholder i set (that i don't forget to change it ;) )